api/proto/teleport/legacy/types/types.proto

```diff
@@ -1062,6 +1064,19 @@ message ProvisionTokenSpecV2CircleCI {
   string OrganizationID = 2 [(gogoproto.jsontag) = "organization_id,omitempty"];
 }
 
+message ProvisionTokenSpecV2Kubernetes {
```

Comment:
nit: the message type should have a comment as well, I only mention this because I think ideally we will eventually enable a linter for it https://github.com/gravitational/teleport/blob/a566bd8018b809a4ed74f0210a35055715845532/api/proto/buf-legacy.yaml#L9-L12

Reply:
done also for github and circleci specs